### PR TITLE
build RPC

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

**Review by yahiheb — APPROVED** [review]
utACK

File: docs/using-wasabi/RPC.md [hunk 1]
```diff
@@ -499,6 +499,26 @@ curl -s --data-binary '{"jsonrpc":"2.0","id":"1","method":"send", "params": { "p
 
 **Note**: error codes are generic and not Wasabi specific.
 
+### build
+
+Builds a transaction.
+It is similar to the send method, except that it will not automatically broadcast the transaction.
+The same commands are available as for the send method.
```
Comment (yahiheb) [inline docs/using-wasabi/RPC.md:506]: Which commands?

Comment (MarnixCroes) [reply docs/using-wasabi/RPC.md:506]: `subtractFee`

Comment (MarnixCroes) [reply docs/using-wasabi/RPC.md]: clarified 79019bb716a67c27089d4bf438b728be97f4bdcb

**Pull request merged**